tests/test_openjdk.py

```diff
@@ -120,6 +120,10 @@ class TestExtendedParams:
                 expected_strings=["test"], environment="ENV1=test"
             ),
         ),
+        (
+            "subprocesses",
+            TestExtendedParams(expected_strings=["tmp", "usr"]),
```

Comment:
Just to be sure: this wants `tmp` and `usr` to be present in the captures output of the command, right?

Reply:
yes 👍